code/week2/operations.cpp

```diff
@@ -0,0 +1,25 @@
+int add(int a, int b) {
+    return a + b;
+}
+
+int subtract(int a, int b) {
+    return add(a, -b);
+}
+
+int multiply(int a, int b) {
+    int accum = 0;
+    for(int i = 0; i < a; i++) {
```

Comment:
This was originally done just to show that you can call a function from inside a function and how chaining them together can produce useful code. Remember that this may be the first time these students are introduced to functions at all and they are expected to following along in the example code as the lectures progresses. We can change the four functions to a one lined multiply with printing the numbers so it is less trivial and an iterative implementation of factorial.